GVFS/GVFS.Platform.Linux/LinuxFileSystemVirtualizer.cs

```diff
@@ -233,13 +233,11 @@ protected override bool TryStart(out string error)
             // because they can only be retrieved from projfs_stop() at present
             if (result != Result.Success)
             {
-                this.Context.Tracer.RelatedError($"{nameof(this.virtualizationInstance.StartVirtualizationInstance)} failed: " + result.ToString("X") + "(" + result.ToString("G") + ")");
-                error = "Failed to start virtualization instance (" + result.ToString() + ")";
+                this.Context.Tracer.RelatedError($"{nameof(this.virtualizationInstance.StartVirtualizationInstance)} failed: {result:X} ({result:G})");
+                error = $"Failed to start virtualization instance ({result})";
                 return false;
```

Comment:
It occurs to me, belatedly, to ask if we could leave these for another day and another PR -- the reason is that this code is just cut-and-pasted from the Mac version, and any extra variations which aren't purely functional make it harder to compare the two.

This won't matter so much once the Linux GVFS provider code is fully merged up into master, and other devs have to make sure to keep things aligned (or refactor into POSIX common code, etc.), but for now, with the ongoing flux in the Mac GVFS provider code, I expect we'll need to go through at least one or two more times and do a diff to make sure the Linux version is up-to-date.  (To put this branch together from my previous draft work, for instance, I took the the latest Mac code, converted `Mac` to `Linux` everywhere, stripped out line ending differences, etc., and then diff'ed everything to make sure all the latest edits were also in the Linux code.)

Reply:
👍  Understood!

Reply:
Fixed, new commit is f82659d3.

Reply:
👍 Thank you!